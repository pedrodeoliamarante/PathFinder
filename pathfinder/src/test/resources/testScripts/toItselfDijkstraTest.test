# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0

FindPath simpleGraph A A

FindPath simpleGraph B B
