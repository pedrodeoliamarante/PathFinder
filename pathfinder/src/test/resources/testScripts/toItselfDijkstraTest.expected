# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph

path from A to A:
A to A with weight 0.000
total cost: 0.000

path from B to B:
B to B with weight 0.000
total cost: 0.000
